subgraphs/_reference_/src/common/metrics.ts

```diff
@@ -0,0 +1,66 @@
+import { BigDecimal, Address, ethereum } from "@graphprotocol/graph-ts";
+import { _Account, _DailyActiveAccount, UsageMetricsDailySnapshot } from "../../generated/schema";
+import { FACTORY_ADDRESS, SECONDS_PER_DAY } from "./constants";
+import { getOrCreateDexAmm, getOrCreatePoolDailySnapshot, getOrCreateUsageMetricSnapshot } from "./getters";
+
+// Update FinancialsDailySnapshots entity
+export function updateFinancials(event: ethereum.Event): void {
```

Comment:
why are they commented out?

Reply:
These are meant more as boilerplates that'll be filled out depending on the subgraph, and will be different from subgraph to subgraph. They are `common` within a subgraph but not common across different subgraphs.

Added comments to explain